huaweicloud/utils/tags.go

```diff
@@ -68,6 +68,21 @@ func TagsToMap(tags []tags.ResourceTag) map[string]string {
 	return result
 }
 
+// FlattenTagsToMap returns the list of tags into a map.
+func FlattenTagsToMap(tags interface{}) map[string]string {
+	if tags == nil {
+		return nil
+	}
+	tagArray := tags.([]interface{})
+	result := make(map[string]string)
+	for _, val := range tagArray {
+		t := val.(map[string]string)
```

Comment:
using `t, ok := val.(map[string]string)` will be better

Reply:
OK, fixed